pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -417,7 +417,8 @@
         "2.0.1",
         "2.1.0",
         "3.0.0",
-        "3.1.0"
+        "3.1.0",
+        "3.1.1"
```

Comment:
I suppose we also want to bump up the baseline below to make sure we use the latest package reference in case somebody depends on this one right?

Reply:
we never changed the baseline before for any of the published packages. is something different about this one ?
cc @ericstj

Reply:
that may be a problem with our process. The baseline version will make sure that if any other package has a package reference to this package, it will force the version to be at least the baseline version (or whichever version has the assembly version in package that you are referencing). You don't always want to do this as you sometimes don't want to pull in many other packages by updating one reference, but for these OOBs it may make sense to update baseline version as well. I'll let @ericstj comment on what the right thing to do here is.

Reply:
Originally POR was to never baseline (even across releases).  We added baselining in 1.1 because folks wanted a consistent wave of packages.  We have only selectively baselined in servicing when we wanted to ensure a dependency is updated as part of updating the depending package, but we've tried to keep our servicing independent.  One thing we could consider was to update baseline for security updates.

Reply:
@ericstj do u mean to bump up the baseline for all the packages which were serviced in security releases or just the package if any with the security fix ?

Reply:
If we ever ship a security fix in package A version X, it makes sense to set the baseline for package A  to version X.

Reply:
okay cool understood. most of our security fixes until now have been in the shared framework